packages/strapi-admin/package.json

```diff
@@ -112,7 +112,7 @@
     "webpack-cli": "^3.3.12",
     "webpack-dev-server": "^3.11.2",
     "webpackbar": "^4.0.0",
-    "yup": "^0.27.0"
+    "yup": "^0.28.2"
```

Comment:
Should we lock the dependency version?

Reply:
I don't think it's needed here since yup is already on v0.32.x so there should be no more updates and npm shouldn't resolve versions higher than 0.28.x